app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,66 @@
+class AnswersController < ApplicationController
+
+  def create
+    @question = Question.find_by(id: params[:question_id])
+    @answer = Answer.new(answer_params)
+    if @answer.save
+      flash.notice = "Answer Created"
+      redirect_to question_path(@question)
+    else
+      redirect_to question_path(@question)
```

Comment:
Maybe render rather than redirect on failure - so that the user's input is not lost.